pwa/src/components/translations/translationForm.tsx

```diff
@@ -2,96 +2,95 @@ import * as React from "react";
 import {
   GenericInputComponent,
   Card,
-  Alert
+  Spinner
 }
   from "@conductionnl/nl-design-system/lib";
-import {Link} from "gatsby";
-import Spinner from "../common/spinner";
-import FlashMessage from 'react-flash-message';
-import {navigate} from "gatsby-link";
+import { Link } from "gatsby";
 import LoadingOverlay from "../loadingOverlay/loadingOverlay";
+import APIService from "../../apiService/apiService";
+import APIContext from "../../apiService/apiContext";
 
 interface TranslationFormProps {
-  id: string,
+  id?: string,
+  tableName?: string
```

Comment:
I think with these changes, we actually have two forms right now. 

1. Creating a new Translation Table and
2. Creating a new Translation

Now, I understand, that both are actually creating the exact same thing in the background. But because we have such a separation in the frontend, I think we should split the forms as well.

So we get a `TranslationTableForm` and a `TranslationForm`.

I'll give you a quick call on how we could maybe choose to keep them in the same file, but still separate them.